### PR TITLE
create release 4.0 branch for licensing operator

### PR DESCRIPTION
**Pull request by SzymonKowalczyk**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Specify your PR reviewers**:

/cc @gyliu513 @morvencao

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: prow/cluster/jobs/IBM/ibm-licensing-operator/IBM.ibm-licensing-operator.release-4.0.yaml [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+        - check
+        image: quay.io/multicloudlab/check-tool:v20200817-b1f2b4c05
+        name: ""
+        securityContext:
+          privileged: true
+    trigger: '(?m)^/test (?:.*? )?check(?: .*?)?$'
+  - name: test_ibm-licensing-operator-amd64
+    cluster: default
+    branches:
+    - ^master$
```
Comment (horis233) [inline prow/cluster/jobs/IBM/ibm-licensing-operator/IBM.ibm-licensing-operator.release-4.0.yaml:24]: @SzymonKowalczyk All the `- ^master$` suppose to be replaced with `- ^release-4.0$` in this file

Comment (SzymonKowalczyk) [reply prow/cluster/jobs/IBM/ibm-licensing-operator/IBM.ibm-licensing-operator.release-4.0.yaml]: Thank tor notice - corrected

**Review by horis233 — APPROVED** [review]
/lgtm

File: prow/cluster/jobs/IBM/ibm-licensing-operator/IBM.ibm-licensing-operator.release-4.0.yaml [hunk 2]
```diff
@@ -0,0 +1,276 @@
+presubmits:
+  IBM/ibm-licensing-operator:
+  - name: check_ibm-licensing-operator
+    cluster: default
+    branches:
+    - ^release-4.0$
```
Comment (ab-ibm) [inline prow/cluster/jobs/IBM/ibm-licensing-operator/IBM.ibm-licensing-operator.release-4.0.yaml:6]: let's use generic one -^release-.*$ so we can use this prow setup for future release branches

**Pull request merged**